### PR TITLE
Update information-barriers-in-teams.md

### PR DESCRIPTION
**Pull request by nirmal-007**
+ Information Barriers feature in Teams does not support federated scenarios today. Removing scenario reference to Federation in the documentation as this sets wrong expectations with admins and customers. IB for guest users remains supported and available.

+ Expanded on the known issues scenarios to make it more clear.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -175,7 +175,9 @@ See the [Information barriers](https://docs.microsoft.com/sharepoint/information
...
+- **Users can't join ad-hoc meetings**: If IB policies are enabled, users are not allowed to join meetings IF the meeting roster size is more than the [meeting attendance limits](limits-specifications-teams.md). The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. Joining a meeting once will add that user to the roster, hence for recurring meetings, the roster fills up fast. Once it reaches the [meeting attendance limits](limits-specifications-teams.md), no additional users are allowed to be added to the meeting chat roster. If IB is enabled for the tenant and chat roster is full for a meeting, new users (those not already on the roster) are not allowed to join the meeting. But if IB is not enabled for the tenant and chat roster is full, new users (those not already on the roster) are allowed to join the meeting, though they won't see the chat option in the meeting. A short term solution is to remove inactive members from the meeting chat roster to make space for new users. We will, however, be increasing the size of meeting chat rosters at a later date.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:178]:
```suggestion
- **Users can't join ad-hoc meetings**: If IB policies are enabled, users are not allowed to join meetings IF the meeting roster size is more than the [meeting attendance limits](limits-specifications-teams.md). The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. Joining a meeting once will add that user to the roster, hence for recurring meetings, the roster fills up fast. Once it reaches the [meeting attendance limits](limits-specifications-teams.md), no additional users are allowed to be added to the meeting chat roster. If IB is enabled for the tenant and the chat roster is full for a meeting, new users (those not already on the roster) are not allowed to join the meeting. But if IB is not enabled for the tenant and the meeting chat roster is full, new users (those not already on the roster) are allowed to join the meeting, though they won't see the chat option in the meeting. A short term solution is to remove inactive members from the meeting chat roster to make space for new users. We will, however, be increasing the size of meeting chat rosters at a later date.
```

Comment (JohanFreelancer9) [reply Teams/information-barriers-in-teams.md:178]: If IB is enabled for the tenant and the chat roster is full for a meeting, new users (those not already on the roster) are not allowed to join the meeting. But if IB is not enabled for the tenant and the meeting chat roster is full, new users (those not already on the roster) are allowed to join the meeting, though they won't see the chat option in the meeting.

Comment (nirmal-007) [reply Teams/information-barriers-in-teams.md:178]: The change looks good.

File: Teams/information-barriers-in-teams.md [hunk 2]
```diff
@@ -175,7 +175,9 @@ See the [Information barriers](https://docs.microsoft.com/sharepoint/information
...
+
+- **Users can't join channel meetings**: If IB policies are enabled, users are not allowed to join channel meetings IF they are not a member of the team. The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. The chat thread in a channel meeting is available to Team/Channel members only, and non-members cannot see/access the chat thread. If IB is enabled for the tenant and a non team member attempts to join a channel meeting, the user is not allowed to join the meeting. But if IB is not enabled for the tenant and a non team member attempts to join a channel meeting, the user is allowed to join the meeting, however, they won't see the chat option in the meeting.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:180]:
```suggestion
- **Users can't join channel meetings**: If IB policies are enabled, users are not allowed to join channel meetings IF they are not a member of the team. The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. The chat thread in a channel meeting is available to Team/Channel members only, and non-members cannot see/access the chat thread. If IB is enabled for the tenant and a non-team member attempts to join a channel meeting, the user is not allowed to join the meeting. But if IB is not enabled for the tenant and a non-team member attempts to join a channel meeting, the user is allowed to join the meeting, however, they won't see the chat option in the meeting.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nirmal-007** [comment]
@scanum - Please help connecting this PR with the technical writer / content publisher so we can get these out. I can share more details / context if required. Thanks.

**Comment by scanum** [comment]
@anwara-ms PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by nirmal-007** [comment]
@anwara-ms - I have expanded on the known issues we have been triaging for a while now. Please take a look and help merge. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by anwara-ms** [comment]
@scanum @JohanFreelancer9  I dont see option to merge. could you do it please?

**Comment by anwara-ms** [comment]
How do I get write access?

**Review by anwara-ms — APPROVED** [review]

**Review by anwara-ms — APPROVED** [review]
looks good. Please publish.

**Comment by nirmal-007** [comment]
@scanum - How can we get this merged? It appears @anwara-ms  is not able to merge.

**Review by anwara-ms — APPROVED** [review]
approved again

**Comment by anwara-ms** [comment]
@scanum Hi. This is urgent. could you please merge this change. I dont have write access.

**Comment by anwara-ms** [comment]
can you please tell me how can I do it myself. I dont have write access. Do PMs have write access, or merge is always done by doc writing team?

**Comment by Kateyanne** [comment]
Hi Anwar – If you make a comment saying it is ready, etc. I can do the merge for you.

Does that help?

Kateyanne

**Pull request merged**